src/main/java/org/zalando/riptide/Router.java

```diff
@@ -105,4 +122,11 @@
         }
     }
 
+    @SafeVarargs
+    public final Router<A> add(Binding<A>... bindings) {
+        Map<A, Binding<A>> map = Router.createMap(asList(bindings));
+        return new Router<A>(selector,
+                Stream.concat(this.bindings.entrySet().stream(), map.entrySet().stream())
```

Comment:
> Nevertheless, we should specify this somewhere that bindings are overwritten based on equality of the attribute.

I thought about this - and nearly implemented it -, but than I realized that we need to include the Executors for proper function, but these will never be simple to compare - only on byte code level. Thus, I skipped this idea and decided that it would be acceptable to copy the router, even if no real change is done.